src/components/Button/button.module.scss

```diff
@@ -398,6 +448,66 @@
     }
 }
 
+.two-state-button {
```

Comment:
Can we move out two state button into its own scss file, its getting crowded here

Reply:
It uses button classes, it's a button. Keeping it here should be necessary.